pkgs/tools/audio/dl-librescore/default.nix

```
@@ -0,0 +1,36 @@
+{ lib
+, buildNpmPackage
+, fetchFromGitHub
+, fetchpatch
+}:
+
+buildNpmPackage rec {
+  pname = "dl-librescore";
+  version = "0.34.47";
+
+  src = fetchFromGitHub {
+    owner = "LibreScore";
+    repo = "dl-librescore";
+    rev = "v${version}";
+    hash = "sha256-yXreyQiIKmZEw2HcpnCW4TxCTHzdq+KuPSlFPFZy2oU=";
+  };
+
+  patches = [
+    # https://github.com/LibreScore/dl-librescore/pull/32
+    (fetchpatch {
+      name = "use-upstream-pdfkit.patch";
+      url = "https://github.com/LibreScore/dl-librescore/commit/e8d9448673fe950df3ed0147be33cd921ed9f9a5.patch";
+      hash = "sha256-MdSv4d7/1W1owMFRFmB55rsf4tTnPy0+cZNap6HrVFc=";
+    })
+  ];
+
+  npmDepsHash = "sha256-XJtJGeTviJNTyrmafaP7qxlOBcbMq9gF14IUWetocyM=";
+
```

```suggestion

  nativeBuildInputs = [ python3 ] ++ lib.optionals stdenv.isDarwin [ darwin.cctools ];

```

Ah, actually this needs `python3` and `libtool` from `darwin.cctools` for the node-gyp build. Surprisingly it only needs this on aarch64-darwin, but at least `python3` should be included on all platforms if it's doing node-gyp stuff...